### PR TITLE
[STAGING] updating gas prices

### PR DESCRIPTION
**Pull request by moda20**

### CODE REVIEW

**Review by magbello157 — APPROVED** [review]

File: src/components/sections/LoginComponent.vue [hunk 1]
```diff
@@ -162,6 +162,7 @@ export default {
           });
         })
         .catch((err) => {
+          console.log(err);
```
Comment (u-hubar) [inline src/components/sections/LoginComponent.vue:165]: Not needed?

File: src/service/contractService.js [hunk 2]
```diff
@@ -34,83 +33,91 @@ class ContractService {
...
+      address,
+      IdentityStorage,
+      this.ethersSigner,
     );
-    const hasPurpose = await IdentityStorageProfile.methods
-      .keyHasPurpose(identity, adminKey, 1)
-      .call({ from: this.web3.eth.defaultAccount });
+    const identity = await IdentityStorageContract.getIdentityId(operationalAddress);
+    const adminKey = this.getAdminKeyFromWallet(adminWalletAddress);
+    const hasPurpose = await IdentityStorageContract.keyHasPurpose(identity, adminKey, 1);
```
Comment (u-hubar) [inline src/service/contractService.js:54]: I guess we can make some constant ``ADMIN_KEY_PURPOSE = 1`` to make it more clear

File: src/service/contractService.js [hunk 3]
```diff
@@ -121,23 +128,10 @@ class ContractService {
...
-    const sanitizedAmount = getAmountWithDecimals(stakeAmountToAdd);
-    await TRACContract.methods.increaseAllowance(stakingContractAddress, sanitizedAmount).send({
-      from: this.web3.eth.defaultAccount,
-    });
-    const stakingContract = new this.web3.eth.Contract(stakingAbi, stakingContractAddress);
-    return await stakingContract.methods.addStake(identityId, sanitizedAmount).send({
-      from: this.web3.eth.defaultAccount,
-    });
-  }
   async addStakeEthers(identityId, stakeAmountToAdd, loadingMessageCallback = null) {
```
Comment (u-hubar) [inline src/service/contractService.js:135]:
```suggestion
  async addStake(identityId, stakeAmountToAdd, loadingMessageCallback = null) {
```

**Review by u-hubar — APPROVED** [review]
Looks good to me. Some small changes needed, not affecting functionality

File: src/service/contractService.js (hunk 3 above)
Comment (moda20) [reply src/service/contractService.js]: updated

File: src/service/contractService.js (hunk 2 above)
Comment (moda20) [reply src/service/contractService.js]: constant added

File: src/components/sections/LoginComponent.vue (hunk 1 above)
Comment (moda20) [reply src/components/sections/LoginComponent.vue]: log statement removed

**Pull request merged**